src/animated/Controller.ts

```diff
@@ -437,6 +450,7 @@ class Controller<DS extends object = any> {
     )
 
     if (changed) {
+      this._willStart(props, oldProps)
```

Comment:
The `onStart` prop is now called for every "update" that adds or changes an animation. Except when all animations are immediate.